src-java/testing/test-library/src/main/java/org/openkilda/testing/service/lockkeeper/LockKeeperVirtualImpl.java

```diff
@@ -83,23 +85,21 @@ public void setController(Switch sw, String controller) {
      * changes are reverted.
      */
     @Override
-    public BlockRequest knockoutSwitch(Switch sw, MultiFloodlightManager manager) {
-        log.debug("Block Floodlight access to switch '{}' by adding iptables rules", sw.getName());
```

Comment:
Why don't you like `log.debug`?)

Reply:
It is misleading since we are not using iptables in virtual implementation, we set dummy controller instead. The setController already has a debug log for changing controller

Reply:
agree